src/main/java/adris/altoclef/util/slots/Slot.java

```diff
@@ -1,74 +1,42 @@
 package adris.altoclef.util.slots;
 
+
 import adris.altoclef.Debug;
-import adris.altoclef.tasks.MineAndCollectTask;
 import net.minecraft.client.MinecraftClient;
 import net.minecraft.client.gui.screen.Screen;
 import net.minecraft.client.gui.screen.ingame.CraftingScreen;
 import net.minecraft.client.gui.screen.ingame.FurnaceScreen;
 import net.minecraft.client.gui.screen.ingame.GenericContainerScreen;
 import net.minecraft.screen.GenericContainerScreenHandler;
-import net.minecraft.screen.ScreenHandler;
+
 
 // Very helpful links
 // Container Window Slots (used to move stuff around all containers, including player):
 //      https://wiki.vg/Inventory
 // Player Inventory Slots (used to grab inventory items only):
 //      https://minecraft.gamepedia.com/Inventory
 public abstract class Slot {
-    private final int _inventorySlot;
-    private final int _windowSlot;
+    private final int inventorySlot;
+    private final int windowSlot;
 
-    private final boolean _isInventory;
+    private final boolean isInventory;
 
-    enum ContainerType {
-        PLAYER,
-        CRAFTING_TABLE,
-        CHEST_SMALL,
-        CHEST_LARGE,
-        FURNACE
-    }
 
     public Slot(int slot, boolean inventory) {
-        _isInventory = inventory;
+        isInventory = inventory;
         if (inventory) {
-            _inventorySlot = slot;
-            _windowSlot = -1;
+            inventorySlot = slot;
+            windowSlot = -1;
             //_windowSlot = inventorySlotToWindowSlot(slot);
         } else {
             //_inventorySlot = windowSlotToInventorySlot(slot);
-            _inventorySlot = -1;
-            _windowSlot = slot;
+            inventorySlot = -1;
+            windowSlot = slot;
         }
     }
 
-    //@SuppressWarnings("CopyConstructorMissesField")
-    /*public Slot(Slot other) {
-        this(other._inventorySlot, true);
-    }*/
-
-    public int getInventorySlot() {
-        if (!_isInventory) {
-            return windowSlotToInventorySlot(_windowSlot);
-        }
-        return _inventorySlot;
-    }
-
-    public int getWindowSlot() {
-        if (_isInventory) {
-            return inventorySlotToWindowSlot(_inventorySlot);
-        }
-        return _windowSlot;
-    }
-
-    public void ensureWindowOpened() {}
-
-    protected abstract int inventorySlotToWindowSlot(int inventorySlot);
-
-    protected abstract int windowSlotToInventorySlot(int windowSlot);
-
     public static Slot getFromInventory(int inventorySlot) {
-        // -1 means cursor.
+        // -1 means cursor. // specify this somewhere, please
```

Comment:
dang you're right, I sort of separate this with "CursorInventorySlot.java" but I should've separated this further down so -1 doesn't appear anywhere except for maybe a constant.